image_weight: 1980
image_height: 1080

image_topic: "/instance_image"